operator-common/src/main/java/io/strimzi/operator/cluster/model/Ca.java

```diff
@@ -174,6 +174,11 @@ public String getSuffix() {
      */
     public static final String ANNO_STRIMZI_IO_CLIENTS_CA_CERT_GENERATION = Annotations.STRIMZI_DOMAIN + "clients-ca-cert-generation";
 
+    /**
+     * Annotation for tracking the Cluster CA key generation trusted by Kubernetes resources
+     */
+    public static final String ANNO_STRIMZI_IO_CLUSTER_CA_KEY_TRUSTED_GENERATION = Annotations.STRIMZI_DOMAIN + "cluster-ca-key-trusted-generation";
```

Comment:
As we discussed on the call => since we add the annotation only when the CA KEY generation is actually used an not just trusted, should we instead get back to just use the `CLUSTER_CA_KEY_GENERATION` annotation?

Reply:
Yeah I meant to do the change and forgot :-(

Reply:
😆